### DISCUSSION TITLE
Windowed (Title less) Slideshow

### DISCUSSION THREAD

**Discussion by SullyPanda76cl** [sylikc/jpegview#174] [Ideas]
Hi... how would everybody think about a windowed slideshow?

At least, I love to have pictures of my loving ones in a small corner of my screen...

**Comment by sylikc** [comment]
I've seen some chatter about people wanting an option to launch without the title bar... Didn't really think it'd be useful, but I can see what the use case is now...

I can add that feature and it can be tied into slideshow

**Comment by sylikc** [comment]
merged with PR https://github.com/sylikc/jpegview/pull/243 ... available in 1.3.46

You should be able to set `WindowBorderlessOnStartup` to `true` and start your slideshow and it should work... let me know if more changes might be needed
